### PR TITLE
Fix errors in README.md

### LINKED ISSUE #1: Fix errors in README.md
There are a few sentences that are to be corrected.

### PR DESCRIPTION
**Pull request by napoco**
The use of personal pronoun “you’ makes the text colloquial and, therefore, lacks professionalism. 
“Once you deploy you will get endpoint” --> “After deployment, the endpoint will be created”
“Use your key and your secret credentials” --> “use key and secret credentials”
“Find your proper region in aws console” --> “find a proper region in AWS console”

Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Use your key and secret credentials; Find your proper region in aws console (def
 
 ### Deploy
 `serverless deploy function --function <function name>`
-Once you deploy you will get endpoint
+After deployment, the endpoint will be acquired
```
Comment (jyleong) [inline README.md]: 'acquired' sounds awkward, how about 'created'

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Set up an AWS IAM serverless-admin with Admin privileges
 
 `> aws configure`
 
-Use your key and secret credentials; Find your proper region in aws console (default us-east-1); Last section set to json
+Use key and secret credentials; Find a proper region in aws console (default us-east-1); Last section set to json
```
Comment (jyleong) [inline README.md]: Good edit, let's capitaize 'aws' to 'AWS'

**Review by jyleong — CHANGES_REQUESTED** [review]
Keep at it. : )

File: README.md (hunk 1 above)
Comment (napoco) [reply README.md]: Oh I see. Alright.

**Review by jyleong — APPROVED** [review]

**Pull request merged**